src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -356,7 +356,25 @@ For example, if you are implementing dark mode support, this event can be used t
 </script>
 ```
 
-## `prefers-reduced-motion`
+## Accessibility
+
+Switching to client-side routing and animating the pages both come with accessibility challenges, and Astro aims to make sites opt-ing into View Transitions as accesible-by-default as possible.
+
+### Route announcement
+
+<Since v="3.2.0" />
+
+The `<ViewTransitions />` component includes a route announcer for page navigation. When using server-side routing assistive technologies will announce the page title after a page loads, which acts as a way to let the user know that the page has changed. When doing client-side routing you want the same thing to occur.
+
+The component achieves this by adding an element to the page with the `aria-live` attribute set to `assertive`. This tells AT (assistive technology) to announce immediately. This announcement text checks these in priority order:
+
+- The `<title>`, if it exists.
+- The first `<h1>` it finds.
+- The `pathname` of the page.
+
+No configuration or action is needed by you to enable this.
```

Comment:
```suggestion
We strongly recommend you always include a `<title>` in each page for accessibility.
```
Suggested to move this existing line up top so people know right away they don't have to read/do anything! Instead, this would be a great place for that note from the implementation PR saying that we suggest always having a title!